config/webpack.config.common.ts

```diff
@@ -0,0 +1,52 @@
+import { Configuration } from "webpack";
+import { join, resolve } from "path";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+
+export const config: Configuration = {
+  entry: resolve("src", "index.tsx"),
+  output: {
+    path: resolve("build"),
+    filename: "[name].[chunkhash].bundle.js",
```

Comment:
Calculating content hash is relatively expensive and there's no point doing that in development mode, so, probably, this particular part should be specific to production mode only.

Reply:
Done